test.disabled/ejabberd_tests/tests/muc_SUITE.erl

```diff
@@ -67,23 +79,48 @@
         items :: [#xmlel{}]
         }).
 
+-define(assertReceivedMatch(Expect), ?assertReceivedMatch(Expect, timer:seconds(5))).
+
+-define(assertReceivedMatch(Expect, Timeout),
```

Comment:
I'm pretty sure we already use this macro somewhere. Maybe it's time to move it to some header file in order not to repeat ourselves.